packages/react/src/drawer/Drawer.css.ts

```diff
@@ -0,0 +1,71 @@
+import { theme } from "../styles";
+import { type RecipeVariants, recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "surface",
+      display: "flex",
+      flexDirection: "column",
+      shadow: "md",
+      z: "popover",
+    },
+    style({
+      position: "fixed",
+    }),
+  ],
+  variants: {
+    position: {
+      bottom: style({
+        borderTop: `1px solid ${theme.colors["border.secondary"]}`,
+        bottom: 0,
+        left: 0,
+        right: 0,
+      }),
+      left: style({
+        borderRight: `1px solid ${theme.colors["border.secondary"]}`,
+        bottom: 0,
+        height: "100%",
+        left: 0,
+        top: 0,
+        width: "25%",
```

Comment:
let's use sprinkle props for width here:

```ts
{
  maxW: ["full", "lg"],
  w: ["full", "3/4"],
}
```

and set it to 75% with a max width of 512px - and make it 100% on mobile